src/hb-open-file-private.hh

```diff
@@ -299,6 +375,7 @@ struct OpenTypeFontFile
     CFFTag		= HB_TAG ('O','T','T','O'), /* OpenType with Postscript outlines */
     TrueTypeTag	= HB_TAG ( 0 , 1 , 0 , 0 ), /* OpenType with TrueType outlines */
     TTCTag		= HB_TAG ('t','t','c','f'), /* TrueType Collection */
+    DFontTag		= HB_TAG ( 0 , 0 , 1 , 0 ), /* Mac Resource Fork */
```

Comment:
It this a fact? It is true about the three remaining macOS dfont files, `/System/Library/Fonts/Courier.dfont`, `/System/Library/Fonts/Geneva.dfont` and ` /System/Library/Fonts/Monaco.dfont` but it seems is not that correct.

Reply:
So maybe we should think about providing a separate API for this if so.